diffus-derive-test/src/lib.rs

```diff
@@ -11,6 +11,65 @@ mod test {
         Diffable,
     };
 
+    #[derive(Diffus, Debug, PartialEq)]
+    struct Pair {
```

Comment:
`Pair` feels a bit generic, maybe something like `Identified` or other, to highlight that it's property is that `Same` is only on its identifier?

Reply:
good point